tests/test_other.py

```diff
@@ -1667,8 +1667,8 @@ def test_dylink_pthread_static_data(self):
       emcc_args=[
         '-pthread', '-Wno-experimental',
         '-s', 'PROXY_TO_PTHREAD',
-        '-s', 'EXIT_RUNTIME=1',
-        '-s', 'MAIN_MODULE=1',
+        '-s', 'EXIT_RUNTIME',
+        '-s', 'MAIN_MODULE=2',
```

Comment:
Looks like a reasonable change, just checking that I'm not missing anything - it's not strictly related to the PR?

Reply:
Reverted.